onchain/multi_tx.go

```diff
@@ -0,0 +1,36 @@
+package onchain
+
+type MultiTxProvider struct {
+	TxProvider
+	Providers []TxProvider
+}
+
+func (m MultiTxProvider) GetRawTransaction(txId string) (hex string, err error) {
+	for _, provider := range m.Providers {
+		hex, err = provider.GetRawTransaction(txId)
+		if err == nil {
+			return hex, nil
+		}
+	}
+	return "", err
+}
+
+func (m MultiTxProvider) BroadcastTransaction(txHex string) (txId string, err error) {
+	for _, provider := range m.Providers {
+		txId, err = provider.BroadcastTransaction(txHex)
```

Comment:
Minor optimization: do it in parallel with an errgroup: https://pkg.go.dev/golang.org/x/sync/errgroup